integrationTests/multiShard/block/executingMiniblocksSc_test.go

```diff
@@ -383,7 +383,7 @@ func proposeBlockWithScTxs(
 			continue
 		}
 
-		body, header, _ := n.ProposeBlock(round)
+		body, header, _ := n.ProposeBlock(round, round)
```

Comment:
can you use separate variable for nonce? otherwise is confusing using round for both round and nonce.
Same for below ProposeBlock usage.

Reply:
Done